### DISCUSSION TITLE
Linux arm64 support

### DISCUSSION THREAD

**Discussion by sarport** [ciderapp/Cider#1636] [Feature Request]
I'm using Ubuntu 22.04 arm64 on Raspberry Pi 4. I tried launch Cider (amd64 Linux version) via emulation tool (box86/box64) but it cannot open since both box64 and box86 doesn't support Chromium/Chromium-based apps (Electron apps like Discord, Spotify, etc.). I can't listen Apple Music on Firefox lack of DRM support on Linux ARM (Actually, there is a way to enable DRM in ARM Chromium). Can you port Cider on ARM (32-bit or 64-bit ARM is not matter for me since I have multiarch armhf & arm64)?

**Comment by thbonk** [comment]
I'm also interested in an arm64 build. I'd like to build a gadget with a Raspberry Pi + Hifi DAC/ADC + Touch Screen. My use case is to attach it to my Hifi Amp in the living room to listen to music and connect BT headphones.

**Reply by thbonk** [reply]
It seems that it's not possible at the moment because of this: https://github.com/castlabs/electron-releases/issues/146 😕

**Reply by sarport** [reply]
The not possible thing is 32-bit ARM port. Good luck with your project but some ARM version is enough to me, 32 or 64-bit doesn't matter me.

**Reply by thbonk** [reply]
@sarport You're right, I overread this detail 🙈

**Comment by kotori2** [comment]
I'm running Kali on Apple M2 (armv8) and I also hope there could be release for ARM64 Linux.

**Comment by GamingLiamStudios** [comment]
We may be able to do a Linux ARM64 release in the future, as there does exist an (un)official DRM implementation from ChromeOS or something, but that would be something in the far future and not currently possible with our current backend for Linux (Electron).

**Comment by monomycelium** [comment]
> It seems that it's not possible at the moment because of this: [castlabs/electron-releases#146](https://github.com/castlabs/electron-releases/issues/146) 😕

The collaborator had mentioned that they will not support the loophole to obtain the dynamic library for the Widevine CDM _directly_ due to "potential legal implications". What if we make the users provide the library, sharing a guide (and disclaimer) to set the loophole up? But it would probably take too much effort just to serve the few of us here…

**Reply by kotori2** [reply]
FYI, there is already an [AUR package for aarch64 Widevine](https://aur.archlinux.org/packages/widevine-aarch64)
